EvolutionModelTests/MockSpecies.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Evolution.Model;
+
+namespace EvolutionModelTests
+{
+    public class MockSpecies : IEater 
```

Comment:
Hier mist nog `IsPredator.Get`. Omdat we deze zelf in willen kunnen stellen is mijn voorstel om `public bool IsPredator { get; set; }` te doen.